bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="broadlink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="command">
+		<item-type>String</item-type>
+		<label>Command</label>
+		<description>The IR/RF command sent to the device</description>
+		<category>Text</category>
+	</channel-type>
+	<channel-type id="temperature">
+		<item-type>Number</item-type>
+		<label>Temperature</label>
+		<category>Temperature</category>
+		<tags>
+			<tag>weather</tag>
```

Comment:
Not sure about the tag weather as it would be inside temperature, weather is usually outside in my interruptation. Why have you added the tag, any reason?

Reply:
Removed - was copy-pasta I think